packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -129,22 +129,24 @@ function Cell(props) {
   }, []);
 
   /**
-   * Handles click event for cell
-   */
-  const onMouseDown = () => {
-    if (!isFocusTrapEnabled) {
-      onCellSelect({ rowId, columnId }, { row: rowIndex, col: columnIndex }, (!isMasked && isSelectable));
-    }
-  };
-
-  /**
-   * Hnadles the onDeactivate callback for FocusTrap component
+   * Handles the onDeactivate callback for FocusTrap component
    */
   const deactiveFocusTrap = () => {
     setFocusTrapEnabled(false);
     columnContext.setCellAriaLiveMessage(intl.formatMessage({ id: 'Terra.worklist-data-grid.resume-navigation' }));
   };
 
+  /**
+   * Handles mouse down event for cell
+   */
+  const onMouseDown = ((event) => {
+    if (!isFocusTrapEnabled) {
+      onCellSelect({
+        rowId, columnId, rowIndex, columnIndex, multiSelect: event.shiftKey, selectedByKeyboard: false, isCellSelectable: (!isMasked && isSelectable),
```

Comment:
Did the linter not ask for this to be formatted differently such as a prop on each line?